internal/config/transport.go

```diff
@@ -61,7 +61,9 @@ func NewTransportConfig(trustedCAPath string) *TransportConfig {
 		TLSHandshakeTimeout:   10 * time.Second,
 		ExpectContinueTimeout: 1 * time.Second,
 		TLSClientConfig: &tls.Config{
-			RootCAs: rootCAs,
+			// Avoid fallback by default to SSL protocols < TLS1.2
+			MinVersion: tls.VersionTLS12,
+			RootCAs:    rootCAs,
```

Comment:
```suggestion
			RootCAs:                  rootCAs,
			PreferServerCipherSuites: true,
			MinVersion:               tls.VersionTLS12,
```

Reply:
This is for gangway as a client, we could consider enabling `PreferServerCipherSuites`. WDYT?

Reply:
I agree :+1:

Reply:
Sounds good :+1: